src/scss/layout/_layout.scss

```diff
@@ -57,35 +43,73 @@ article {
 			display: table;
 		}
 	}
-}
 
-/*
- * Default WP Alignment Classes
+	/*
+	WP selector's customized styles for the WeCount website
  *****************************************************************************/
-figcaption {
-	display: block;
-	font-size: rem(14);
-	font-style: italic;
-	padding: 1rem 0;
-	text-align: center;
+	figure {
+		display: table;
+		margin-left: auto;
+		margin-right: auto;
+	}
+
+	.wp-block-image {
+		img {
+			height: 100%;
+			max-width: 100%;
+		}
+
+		figcaption {
+			caption-side: bottom;
+			display: table-caption;
+			font-size: rem(14);
+			font-style: italic;
+			padding: 1rem 0;
+			text-align: center;
+		}
+
+		.alignleft {
+			float: left;
+			margin-right: rem(25);
+			margin-top: 0;
+		}
+
+		.alignright {
+			float: right;
+			margin-left: rem(25);
+			margin-top: 0;
+		}
+	}
+
+	.wp-block-image figure .alignfull {
+		margin-left: calc(-100vw / 2 + 100% / 2);
+		margin-right: calc(-100vw / 2 + 100% / 2);
+		max-width: 100vw;
+	}
+
+	.wp-block-image figure .alignwide {
+		margin-left: rem(-100);
+		margin-right: rem(-100);
+		max-width: rem(1024);
```

Comment:
I wonder why `max-width` is set to a fix `1024px` value rather than a relative value such as `100%`? Same question for line 97.

Reply:
Good catch! The value for these lines should be `100%`.